src/components/CommentCard.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {ClockSVG} from './svgs/ClockSVG';
+import {HeartSVG} from './svgs/HeartSVG';
+
+export const CommentCard = ({ authorName, date, time, numHearts, content }) => {
+    return (
+        <div className='component-to-copy gb-comment-card'>
+            <div className='user-avatar-wrapper'>
+                <a href='#' className='gb-avatar gb-avatar-large user-avatar'/>
+            </div>
+            <div className='comment-content-wrapper'>
+                <div className='top-content'>
+                    <a href='#' className='author-name gb-title-tiny gb-text-black'><b>{authorName}</b></a>
+                    <div className="gb-card-label comment-label">
```

Comment:
You can take the label out , make it's own component, probably we are going to use them in other places too